File: woodroid/src/main/java/me/gilo/woodroid/models/filters/Role.java
package me.gilo.woodroid.models.filters;

public enum Role {
    ALL{
        @Override
        public String toString() {
            return "all";
        }
    },
    ADMINISTRATOR{
        @Override
        public String toString() {
            return "administrator";
        }
    },
    EDITOR{
        @Override
        public String toString() {
            return "editor";
        }
    },

    AUTHOR{
        @Override
        public String toString() {
            return "author";
        }
    },

    CONTRIBUTOR{
        @Override
        public String toString() {
            return "contributor";
        }
    },

    SUBSCRIBER{
        @Override
        public String toString() {
            return "subscriber";
        }
    },

    CUSTOMER{
        @Override
        public String toString() {
            return "customer";
        }
    },

    SHOP_MANAGER{
        @Override
        public String toString() {
            return "shop_manager";
        }
    },
}

File: core/src/main/java/me/gilo/woodroid/core/models/filters/Role.kt
package me.gilo.woodroid.models.filters

enum class Role {
    ALL {
        override fun toString(): String {
            return "all"
        }
    },
    ADMINISTRATOR {
        override fun toString(): String {
            return "administrator"
        }
    },
    EDITOR {
        override fun toString(): String {
            return "editor"
        }
    },

    AUTHOR {
        override fun toString(): String {
            return "author"
        }
    },

    CONTRIBUTOR {
        override fun toString(): String {
            return "contributor"
        }
    },

    SUBSCRIBER {
        override fun toString(): String {
            return "subscriber"
        }
    },

    CUSTOMER {
        override fun toString(): String {
            return "customer"
        }
    },

    SHOP_MANAGER {
        override fun toString(): String {
            return "shop_manager"
        }
    }
}
